Homework/src/io/codelex/arithmetic/practice/ExcerciseFour.java

```diff
@@ -0,0 +1,14 @@
+package io.codelex.arithmetic.practice;
+
+//So does it have to compute !n or !10 ?
+
+public class ExcerciseFour {
+    public static void main(String[] args) {
+        System.out.println(Product1ToN());
+    }
+    static int Product1ToN(){
+        int sum=1;
+        for(int i = 1; i<=10; i++)sum*=i;
```

Comment:
Javā jālieto figūriekavas pēc for. Šis tehniski strādā, bet par to tiktu aizrādīts arī darba vietā.